src/python/setup.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import os
+
+from setuptools import find_packages, setup
+
+
+install_requires = open("requirements.txt").read().strip().split("\n")
+
+setup(
+    name="aqueduct-ml",
+    version="0.0.1",
```

Comment:
Is this something that should ever be changed, or is it supposed to be hard-coded?

Reply:
We should bump this when we are ready to cut another release.